coalib/parsing/filters/Filter.py

```diff
@@ -0,0 +1,15 @@
+def filter(filter_applier):
+    def filter_wrapper(bears, args):
```

Comment:
use `functools.wraps` to make the old function signature appear for debugging purposes